services/spar/src/Spar/SCIM.hs

```diff
@@ -114,7 +120,24 @@ apiScim = hoistSCIM (toServant (SCIM.siteServer configuration))
 ----------------------------------------------------------------------------
 -- UserDB
 
--- $mapping
+-- | Retrieve 'IdP' from 'ScimTokenInfo' and call 'validateSCIMUser''.
+validateSCIMUser
+  :: forall m. (m ~ SCIM.SCIMHandler Spar)
```

Comment:
or tell me to quit it.  :-)

(i'm mostly doing that to indicate that i would prefer this to be a set of constraints.  i'm happy to keep doing it, but i'm not as emotionally attached to the habit as to indentation...)

Reply:
> i'm mostly doing that to indicate that i would prefer this to be a set of constraints

Okay, then I'll try to do the same. Not sure it will catch on but I don't mind.